[ismael@sourcemage.org: Stripped-out changes that don't make a difference
 to the installation]

Author: Andreas Beckmann <anbe@debian.org>
Description: fix typos found by Lintian

--- a/libexplain/buffer/errno/acl_set_file.c
+++ b/libexplain/buffer/errno/acl_set_file.c
@@ -224,7 +224,7 @@ explain_buffer_errno_acl_set_file_explan
          * new ACL cannot be extended or the file system is out
          * of file allocation resources.
          */
-        explain_buffer_enospc(sb, pathname, "pathnme");
+        explain_buffer_enospc(sb, pathname, "pathname");
         break;
 
     case ENOTDIR:
--- a/libexplain/buffer/errno/mount.c
+++ b/libexplain/buffer/errno/mount.c
@@ -540,7 +540,7 @@ explain_buffer_errno_mount_explanation(e
             (
                 sb,
                 /* FIXME: i18n */
-                "target is still busy with open files or working diretories "
+                "target is still busy with open files or working directories "
             );
             return;
         }
--- a/libexplain/buffer/errno/read.c
+++ b/libexplain/buffer/errno/read.c
@@ -159,7 +159,7 @@ explain_buffer_errno_read_explanation(ex
                 (
                     sb,
                     /* FIXME: i18n */
-                    "the file desriptor is attached to an object "
+                    "the file descriptor is attached to an object "
                     "which is unsuitable for reading; or, the file was "
                     "opened with the O_DIRECT flag, and either the "
                     "address specified in data, the value specified "
--- a/man/man1/explain.1
+++ b/man/man1/explain.1
@@ -41,7 +41,7 @@ explain \- explain system call error mes
 .SH DESCRIPTION
 The \*(n) command is used to decode an error return read from an
 \f[I]strace\fP(1) listing, or silimar.  Because this is being
-deciphered in a different process than the orginal, the results
+deciphered in a different process than the original, the results
 will be less accurate than if the program itself were to use
 \f[I]libexplain\fP(3).
 .SS Functions
--- a/man/man1/explain_lca2010.1
+++ b/man/man1/explain_lca2010.1
@@ -988,7 +988,7 @@ tries to, you can't open a file in \f[CW
 .E(
 open_max = sysconf(_SC_OPEN_MAX);
 .E)
-This one wan't so difficult, there is a \f[I]sysconf\fP(3) way of
+This one wasn't so difficult, there is a \f[I]sysconf\fP(3) way of
 obtaining the limit.
 .SS ENFILE, Too many open files in system
 This error occurs when the system limit on the total number of open files
@@ -1242,7 +1242,7 @@ what \f[I]errno\fP(3) values may be retu
 \[bu]
 the cause of each error.
 .PP
-Because of the OO nature of function call dispatching withing the
+Because of the OO nature of function call dispatching within the
 kernel, you need to read \f[I]all\fP sources implementing that
 \f[I]ioctl\fP(2) request, not just the generic implementation.
 It is to be expected that different kernels will have different error
--- a/man/man3/explain.3
+++ b/man/man3/explain.3
@@ -137,8 +137,8 @@ Default: 0
 .TP 8n
 internal\[hy]strerror
 This option controls the source of
-system eror message texts.
-If false, it uses \f[I]streror\P(3) for the text.
+system error message texts.
+If false, it uses \f[I]strerror\P(3) for the text.
 If true, it uses internal string for the text.
 This is mostly of use for automated testing, to avoid false negatives
 induced by inconsistencies across Unix implementations.
--- a/man/man3/explain_chmod_or_die.3
+++ b/man/man3/explain_chmod_or_die.3
@@ -31,7 +31,7 @@ explain_chmod_or_die \- change permissio
 void explain_chmod_or_die(const char *pathname, int mode);
 .SH DESCRIPTION
 The explain_chmod_or_die function is used to call the
-\f[I]chmod\fP(2) system call.  On failure an explanation wiil be printed
+\f[I]chmod\fP(2) system call.  On failure an explanation will be printed
 to stderr, obtained from \f[I]explain_chmod\fP(3), and the the
 process terminates by calling \f[CW]exit(EXIT_FAILURE)\fP.
 .PP
--- a/man/man3/explain_lseek.3
+++ b/man/man3/explain_lseek.3
@@ -233,7 +233,7 @@ value of \f[I]errno\fP.
 The original fildes, exactly as passed to the \f[I]lseek\fP(2) system call.
 .TP 8n
 \f[I]offset\fP
-The orginal offset,
+The original offset,
 exactly as passed to the \f[I]lseek\fP(2) system call.
 .TP 8n
 \f[I]whence\fP
--- a/man/man3/explain_write.3
+++ b/man/man3/explain_write.3
@@ -121,7 +121,7 @@ explained and this function, because man
 value of \f[I]errno\fP.
 .TP 8n
 \f[I]fildes\fP
-The orginal fildes, exactly as passed to the \f[I]write\fP(2) system call.
+The original fildes, exactly as passed to the \f[I]write\fP(2) system call.
 .TP 8n
 \f[I]data\fP
 The original data, exactly as passed to the \f[I]write\fP(2) system call.
